src/mappers/station-to-dropdown-object.ts

```diff
@@ -0,0 +1,25 @@
+import { Station } from '../models/station';
+import { TAutocompleteDropdownItem } from 'react-native-autocomplete-dropdown';
+
+export function stationToDropdownObject(
+  station: Station,
+  index: number,
+): TAutocompleteDropdownItem {
+  return {
+    id: index.toString(),
+    title: `${station.stationName} (${station.crs})`,
+  };
+}
+
+export function dropdownObjectToStation(
+  dropdownObject: TAutocompleteDropdownItem,
+): Station {
+  if (dropdownObject && dropdownObject.title) {
+    const titleParse = /([a-zA-Z ]+) \(([A-Z]{3})\)/;
```

Comment:
It's not super clear what this variable is from the name - I can kind of work it out, but it might be nicer if it was a bit more descriptive